pyxem/utils/general_utils.py

```diff
@@ -0,0 +1,20 @@
+from skimage.filters import gaussian
+import numpy as np
+
+
+def get_direct_beam_center(image, window=60, window_shift=(0, 0), **kwargs):
```

Comment:
Pretty sure this can be replaced with https://github.com/pyxem/pyxem/blob/d6e3194bab5d3f0071e062818f301251d8ac68c2/pyxem/utils/expt_utils.py#L644

Reply:
will look into it